neqo-transport/src/connection.rs

```diff
@@ -1420,6 +1420,38 @@ impl Connection {
         }
     }
 
+    fn validate_active_connection_id_limit(&self) -> Res<()> {
+        if let Some(path) = &self.path {
+            let tph = self.tps.borrow();
+            let remote = tph.remote();
+            if path.remote_cid.len() == 0 {
```

Comment:
`is_empty()` should work here.  I'm surprised that clippy didn't object.

Reply:
It doesn't surprise me -- clippy and I are in a fight right now. For some reason I can't get it to consistently run in a way that I am confident it is actually doing any checking. Thanks for the note on this. I will definitely fix it.